temba/channels/models.py

```diff
@@ -2878,9 +2903,9 @@ class Alert(SmartModel):
     TYPE_POWER = 'P'
     TYPE_SMS = 'S'
 
-    TYPE_CHOICES = ((TYPE_POWER, _("Power")),                 # channel has low power
-                    (TYPE_DISCONNECTED, _("Disconnected")),   # channel hasn't synced in a while
-                    (TYPE_SMS, _("SMS")))                     # channel has many unsent messages
+    TYPE_CHOICES = ((TYPE_POWER, _("Power")),  # channel has low power
+                    (TYPE_DISCONNECTED, _("Disconnected")),  # channel hasn't synced in a while
+                    (TYPE_SMS, _("SMS")))  # channel has many unsent messages
```

Comment:
Please restore all these comments to what they were. (and in other places) It confuses the diff